### PR TITLE
KOMENCI_VERSION env var got lost in multi-stage docker build

### PR DESCRIPTION
**Pull request by bowd**
Title says it all

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,6 @@
 FROM node:10 as builder
 
 ARG KOMENCI_VERSION
```
Comment (gnardini) [inline Dockerfile:3]: Is this first `ARG KOMENCI_VERSION` needed?

**Review by gnardini — APPROVED** [review]
Small question, looks great :)

File: Dockerfile (hunk 1 above)
Comment (bowd) [reply Dockerfile:3]: I'm actually not sure, hmmm let's try without it and see if it works 😂

Comment (gnardini) [reply Dockerfile]: Just tested this in alfajores and it worked well :) Merging this PR

**Pull request merged**